boards/xtensa/up_squared_adsp/tools/adsplog.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/python3
```

Comment:
I would send this and 3f774d989a425cb6a55d5680c9afb5e6f2395d90 already upstream.